Andlet/Andlet/Models/TestFirestoreConnectionView.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+import FirebaseFirestore
+
+struct TestFirestoreConnectionView: View {
```

Comment:
Should we keep this view in the Models folder ?

Reply:
Ohh no, It was just a test